control_center.yml

```diff
@@ -9,7 +9,7 @@
         name: confluent.variables
 
     - name: Check if Control Center Service Running
-      shell: "systemctl show -p SubState {{control_center_service_name}}"
+      command: "systemctl show -p SubState {{control_center_service_name}}" # noqa command-instead-of-module
```

Comment:
I've been thinking, instead of suppressing this linting error, should this instead be the `service_facts` module?

Reply:
That is a very good idea, but my initial work on that shows that vi might have issues with the `.service` postfix. On my Ubuntu laptop all services are named just <service-name>, while on our RHEL servers the postfix `.service` is added to all service names??? Even if both are based on systemd. I do not understand why, and that will complicate our work on this.

Reply:
Hmm, I've found `systemctl status confluent-server.service` and `systemctl status confluent-server` both work on rhel... so I wonder if we always add the .service and be ok.

Or a if statement, or a regex replace filter could be applied. i have ubuntu and centos molecule scenarios that I can test w.

Reply:
I have not tried postfixing with `.service`. Please check if it works ok.

Reply:
Just tested this on debian containers:
```
    - name: Populate service facts
      service_facts:

    - debug:
        msg: "{{ ansible_facts.services[zookeeper_service_name + '.service'] }}"
```

It returned:
```
TASK [debug] *******************************************************************
ok: [zookeeper1] => {
    "msg": {
        "name": "confluent-zookeeper.service",
        "source": "systemd",
        "state": "running",
        "status": "enabled"
    }
}
```

So I think we are good to proceed with this change!

Reply:
I also tested the defaulting logic works on debian as well (ie no packages installed and the service_state var gets set as 
```
TASK [debug] *******************************************************************
ok: [zookeeper1] => {
    "service_state": "unknown"
}
```

Reply:
tested on ubuntu as well. `service_facts` works great